.github/workflows/build_images.yaml

```diff
@@ -5,6 +5,7 @@ on:
   schedule: [{cron: "14 14 * * TUE"}] # Every Tuesday at 14:14
 jobs:
   setup:
+    if: github.event_name != 'pull_request' || !contains('OWNER,MEMBER,COLLABORATOR', github.event.pull_request.author_association)
```

Comment:
Alas, this will stop it building `master` on a schedule, I was thinking something like this:
https://github.com/stfc/SCD-OpenStack-Utils/blob/master/.github/workflows/rabbit_consumer_chart.yaml#L5